controllers/dns_helper.go

```diff
@@ -232,7 +231,7 @@ func (dh *dnsHelper) getLoadBalancedEndpoints(mcgTarget *multicluster.GatewayTar
 			}
 
 			if len(ipValues) > 0 {
-				clusterLbName := strings.ToLower(fmt.Sprintf("%s.%s", cgwTarget.GetShortCode(), lbName))
+				clusterLbName := strings.ToLower(fmt.Sprintf("%s-%s.%s", mcgTarget.ClusterName, mcgTarget.GetShortCode(), lbName))
```

Comment:
Should this not be `cgwTarget.ClusterName`? mcgTarget has many cluster targets each with a unique name (id). This will likely not work with ocm.